gerrit-entrypoint.sh

```diff
@@ -18,7 +22,7 @@ if [ "$1" = "/gerrit-start.sh" ]; then
 
   if [ -z "$(ls -A "$GERRIT_SITE")" ]; then
     echo "First time initialize gerrit..."
-    gosu ${GERRIT_USER} java -jar "${GERRIT_WAR}" init --batch --no-auto-start -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
+    gosu ${GERRIT_USER} java ${JAVA_OPTIONS} ${JAVA_MEM_OPTIONS} -jar "${GERRIT_WAR}" init --batch --no-auto-start -d "${GERRIT_SITE}" ${GERRIT_INIT_ARGS}
```

Comment:
The reason I used JAVA_HEAPLIMIT is because it is already an option used by Gerrit and the configuration script: https://github.com/openfrontier/docker-gerrit/blob/master/gerrit-entrypoint.sh#L129 and https://gerrit-review.googlesource.com/Documentation/config-gerrit.html#container.heapLimit

So people have likely already configured their container with an env var JAVA_HEAPLIMIT=6g for instance, and the purpose of this change was to possibly reuse that.
There is nothing preventing people from doing JAVA_OPTIONS="-Xmx2g", although I'm not sure how Gerrit would handle that since it has a dedicated config option.